### PR TITLE
feat: Coverage

### PR DESCRIPTION
**Pull request by zippy1978**
- [x] Implement a CoverageParser
- [x] Implement a CoverageSensor

References
- https://github.com/insideapp-oss/sonar-flutter/tree/develop/sonar-flutter-plugin/src/main/java/fr/insideapp/sonarqube/flutter/coverage

Dod
- [x] Unit tests
- [x] Documentation (and developer documentation) update

### CODE REVIEW

File: commons/src/test/resources/coverage/cobertura.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+<!DOCTYPE coverage SYSTEM "http://cobertura.sourceforge.net/xml/coverage-04.dtd">
+<coverage line-rate="0.7692307692307693" branch-rate="0.8000000000000000" lines-covered="10" lines-valid="13" branches-covered="4" branches-valid="5" complexity="0.0" timestamp="1650561550" version="Slather 2.7.2">
+  <sources>
+    <source>/Users/ggrousset/Desktop/SQApp</source>
```
Comment (JessyB) [inline commons/src/test/resources/coverage/cobertura.xml:24]: Ca pointe vers un dossier local de ta machine

Comment (zippy1978) [reply commons/src/test/resources/coverage/cobertura.xml:24]: Right,
However, it does not matter at all (it's a sample cobertura report to unit test parsing).
But I'll fix it anyway !

Comment (zippy1978) [reply commons/src/test/resources/coverage/cobertura.xml]: @JessyB done !

**Review by JessyB — APPROVED** [review]
En fait le package dans sonar-apple-plugin a une faute.
fr.insideapp.sonaqube -> fr.insideapp.sonarqube

**Pull request merged**